### DISCUSSION TITLE
Failure processing message with error 80004005 when accepting a change of an event

### DISCUSSION THREAD

**Discussion by walterzanette** [kewisch/gdata-provider#452] [Support]
If I accept the modification of an event already in my calendar I can see the following error in the console:

Calendar: [calGoogleCalendar] Modifying item ############ Progress Update failed:2147500037: {
 "error": {
  "errors": [
   {
    "domain": "calendar",
    "reason": "disallowedFieldMutation",
    "message": "Insufficient permissions to modify a field on the resource."
   }
  ],
  "code": 403,
  "message": "Insufficient permissions to modify a field on the resource."
 }
}

The message bos "Elaborazione del messaggio fallita. Stato: 80004005" appears.

There seems to be no reason for this since the calendar is mine and I can change the event if working from the smartphone or from the browser.

Any idea?

Thanks!

**Comment by henning** [comment]
I already started a discussion about that:
https://github.com/kewisch/gdata-provider/discussions/444 

Which I also made an issue of:
https://github.com/kewisch/gdata-provider/issues/449

Maybe you want to add your information to that one ;)

**Comment by walterzanette** [comment]
Hi Henning,

I added the info and a snapshot of the error console. It seems somehow 
related to a "disallowedFieldMutation" error that comes back from the 
server. This happens only if the event was sent from Teams. If it is 
sent from google calendar all works as expected.

In the meantime: thanks for this awesome plugin!!

Best,

------------------------------------------------------------------------
**

**

*Walter Zanette*
Euclid Labs Srl*
Tel*+39-0422-887075 | *Mobile* +39-393-8597671
***@***.*** ***@***.***>
*www.euclidlabs.it* <http://www.euclidlabs.it/>
*
* <http://www.euclidlabs.it/>
Il 03/06/2022 20:41, Henning Sprang ha scritto:
